src/test/java/com/example/amazoncom/amazontest.java

```diff
@@ -7,51 +7,38 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import java.time.Duration;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class amazontest {
```

Comment:
Pls use the class name in accordance with Java naming conventions
can be found in learn materials: https://elearn.epam.com/assets/courseware/v1/1464695e99aa39237d39ac508e7672cc/asset-v1:RD_CEE+TAJavaOP+1+type@asset+block/solid_kiss_dry_pattetns_pageobject.pptx
or on the official page: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html